ide/plugins/java-format/src/content/formats/webdriver-testng.js

```diff
@@ -0,0 +1,523 @@
+}
+
+function formatComment(comment) {
+  return comment.comment.replace(/.+/mg, function(str) {
+    return "// " + str;
+  });
+}
+
+function keyVariable(key) {
+  return "Keys." + key;
+}
+
+this.sendKeysMaping = {
+  BKSP: "BACK_SPACE",
+  BACKSPACE: "BACK_SPACE",
+  TAB: "TAB",
+  ENTER: "ENTER",
+  SHIFT: "SHIFT",
+  CONTROL: "CONTROL",
+  CTRL: "CONTROL",
+  ALT: "ALT",
+  PAUSE: "PAUSE",
+  ESCAPE: "ESCAPE",
+  ESC: "ESCAPE",
+  SPACE: "SPACE",
+  PAGE_UP: "PAGE_UP",
+  PGUP: "PAGE_UP",
+  PAGE_DOWN: "PAGE_DOWN",
+  PGDN: "PAGE_DOWN",
+  END: "END",
+  HOME: "HOME",
+  LEFT: "LEFT",
+  UP: "UP",
+  RIGHT: "RIGHT",
+  DOWN: "DOWN",
+  INSERT: "INSERT",
+  INS: "INSERT",
+  DELETE: "DELETE",
+  DEL: "DELETE",
+  SEMICOLON: "SEMICOLON",
+  EQUALS: "EQUALS",
+
+  NUMPAD0: "NUMPAD0",
+  N0: "NUMPAD0",
+  NUMPAD1: "NUMPAD1",
+  N1: "NUMPAD1",
+  NUMPAD2: "NUMPAD2",
+  N2: "NUMPAD2",
+  NUMPAD3: "NUMPAD3",
+  N3: "NUMPAD3",
+  NUMPAD4: "NUMPAD4",
+  N4: "NUMPAD4",
+  NUMPAD5: "NUMPAD5",
+  N5: "NUMPAD5",
+  NUMPAD6: "NUMPAD6",
+  N6: "NUMPAD6",
+  NUMPAD7: "NUMPAD7",
+  N7: "NUMPAD7",
+  NUMPAD8: "NUMPAD8",
+  N8: "NUMPAD8",
+  NUMPAD9: "NUMPAD9",
+  N9: "NUMPAD9",
+  MULTIPLY: "MULTIPLY",
+  MUL: "MULTIPLY",
+  ADD: "ADD",
+  PLUS: "ADD",
+  SEPARATOR: "SEPARATOR",
+  SEP: "SEPARATOR",
+  SUBTRACT: "SUBTRACT",
+  MINUS: "SUBTRACT",
+  DECIMAL: "DECIMAL",
+  PERIOD: "DECIMAL",
+  DIVIDE: "DIVIDE",
+  DIV: "DIVIDE",
+
+  F1: "F1",
+  F2: "F2",
+  F3: "F3",
+  F4: "F4",
+  F5: "F5",
+  F6: "F6",
+  F7: "F7",
+  F8: "F8",
+  F9: "F9",
+  F10: "F10",
+  F11: "F11",
+  F12: "F12",
+
+  META: "META",
+  COMMAND: "COMMAND"
+};
+
+/**
+ * Returns a string representing the suite for this formatter language.
+ *
+ * @param testSuite  the suite to format
+ * @param filename   the file the formatted suite will be saved as
+ */
+function formatSuite(testSuite, filename) {
+  var suiteClass = /^(\w+)/.exec(filename)[1];
+  suiteClass = suiteClass[0].toUpperCase() + suiteClass.substring(1);
+
+  var formattedSuite = '<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd" >\n\n'
+      + '<!--testng.xml-->\n'
+      + '<!--For more details please visit: http://testng.org/doc/documentation-main.html-->\n'
+      + '<suite name="Suite' + suiteClass + '" verbose="1" >\n'
+      + indents(1) + '<test name="' + suiteClass + '">\n'
+      + indents(2) + '<classes>\n'
+
+  for (var i = 0; i < testSuite.tests.length; ++i) {
+    var testClass = testSuite.tests[i].getTitle();
+    formattedSuite += indents(3)
+        + '<class name="com.example.tests.'+ testClass +'"/>\n';
+  }
+
+  formattedSuite += indents(2) + "</classes>\n"
+      + indents(1) + "</test>\n"
+      + "</suite>";
+
+  return formattedSuite;
+}
+
+function defaultExtension() {
+  return this.options.defaultExtension;
+}
+
+this.options = {
+  receiver: "driver",
+  packageName: "com.example.tests",
+  indent:    '2',
+  initialIndents:    '2',
```

Comment:
Fixed in [#7e0de8f](https://github.com/rasika90/selenium/commit/7e0de8f7253c45ace8d0fd539f4712e1f91349c1).